src/core/css/SearchResult.scss

```diff
@@ -73,11 +81,4 @@
 .SearchResult-actions {
   align-items: center;
   display: flex;
-}
-
-.SearchResult-icon {
-  display: inline-block;
-  float: right;
-  height: 38px;
-  width: 38px;
-}
+}
```

Comment:
I changed the width of SearchResult-text to px and added media queries to make it responsive.